### ISSUE TITLE
Honor `org-agenda-start-on-weekday`

### ISSUE THREAD

**Issue by dato** [Fuco1/org-clock-budget#2]
It would be nice if the budget report would use the users’ idea of when a week starts (instead of Monday).

I, for example, plan my weeks starting on Saturday.

org-mode has a variable for this, `org-agenda-start-on-weekday`:

> Non-nil means start the overview always on the specified weekday. 0 denotes Sunday, 1 denotes Monday, etc.

I _think_ it should be fine to use it for org-clock-budget-report. I will try to submit a patch if I can.

Btw thanks for the library, it suits my needs fine!

**Comment by Fuco1** [comment]
You are right, we should respect the setting.  I had no idea it existed though :D Should be doable.
